apps/condo/domains/resident/tasks/helpers/sendBillingReceiptsAddedNotifications.spec.js

```diff
@@ -208,4 +210,38 @@ describe('sendBillingReceiptsAddedNotificationsForPeriod', () => {
         })
     })
 
+    describe('Real-life test cases', () => {
```

Comment:
That's too many match changes and only one test. You should think deeply about edge cases

Reply:
The business logic of the task has not changed. The moments of implementation have changed. Previously, the starting point was `BillingReceipt`, now `BillingContext`. Therefore, the old tests check the logic in the same way as before. The real-life test was added only to check the launch of the task after the `registerBillingReciept` mutation to check the full flow. Old tests remain relevant